src/core/server/saved_objects/migrations/core/unused_types.ts

```diff
@@ -41,31 +41,35 @@ export const REMOVED_TYPES: string[] = [
 // saved objects which are no longer used. These saved objects will still be
 // kept in the outdated index for backup purposes, but won't be available in
 // the upgraded index.
-export const excludeUnusedTypesQuery: estypes.QueryDslQueryContainer = {
+export const excludeUnusedTypesQuery: QueryDslQueryContainer = {
   bool: {
-    must_not: [
-      ...REMOVED_TYPES.map((typeName) => ({
-        term: {
-          type: typeName,
-        },
-      })),
-      // https://github.com/elastic/kibana/issues/96131
-      {
-        bool: {
-          must: [
-            {
-              match: {
-                type: 'search-session',
-              },
+    filter: {
+      bool: {
+        must_not: [
```

Comment:
I am placing this `must_not` under a `bool.filter` to avoid unnecessary score calculations.

Reply:
Oh, didn't know that! thanks for the info. Of course I will revert it! 😃